cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/DefaultMetricDatasetFactory.java

```diff
@@ -89,18 +83,7 @@ public FactTable getOrCreateFactTable(int resolution) {
     // for efficient counters
     props.setReadlessIncrementSupport(true);
```

Comment:
should we remove this from this class now that we set it also in `HBaseMetricsTableDefinition.java`?

Reply:
Sure. This class was so badly written...